pootle/apps/pootle_app/management/commands/create_project.py

```diff
@@ -0,0 +1,148 @@
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand
+from django.core.exceptions import ValidationError
+from django.core.management import CommandError
+from django.core.validators import validate_email
+
+from pootle.core.delegate import formats
+from pootle_fs.delegate import fs_plugins
+from pootle_fs.presets import FS_PRESETS
+from pootle_language.models import Language
+from pootle_project.models import Project, PROJECT_CHECKERS
+
+
+class Command(BaseCommand):
+    help = "Creates a new project from command line."
+
+    def add_arguments(self, parser):
+        format_names = formats.get().keys()
+        fs_plugins_names = fs_plugins.gather().keys()
+
+        parser.add_argument(
+            'code',
+            action='store',
+            help='Project code',
+        )
+        parser.add_argument(
+            "--name",
+            action="store",
+            dest="name",
+            help="Project name",
+        )
+        parser.add_argument(
+            "--filetype",
+            action="append",
+            dest="filetypes",
+            choices=formats.get().keys(),
+            default=[],
+            help="File types: {0}. Default: po".format(
+                 " | ".join(format_names)),
+        )
+        mapping_group = parser.add_mutually_exclusive_group(required=True)
+        mapping_group.add_argument(
+            "--preset-mapping",
+            action="store",
+            dest="preset_mapping",
+            choices=FS_PRESETS.keys(),
+            help="Preset filesystem layout: {0}".format(
+                 " | ".join(FS_PRESETS.keys())),
+        )
+        mapping_group.add_argument(
+            "--mapping",
+            action="store",
+            dest="mapping",
+            help="Custom filesystem layout",
+        )
+        parser.add_argument(
+            "--fs-type",
+            action="store",
+            dest="fs_type",
+            default="localfs",
+            choices=fs_plugins.gather().keys(),
+            help="Filesystem type: {0}".format(
+                " | ".join(fs_plugins_names)),
+        )
+        parser.add_argument(
+            "--fs-url",
+            action="store",
+            dest="fs_url",
+            default="",
+            help="Filesystem path or URL.",
+        )
+        parser.add_argument(
+            "--source-language",
+            action="store",
+            dest="sourcelang",
+            default="en",
+            help="Source language. Examples: [en | es | fr | ...].\
+                     Default: %(default)s",
+        )
+        parser.add_argument(
+            "--report-email",
+            action="store",
+            dest="contact",
+            help="Contact email for reports. Example: admin@mail.com. Default: ",
```

Comment:
There is no default here.

Reply:
You told to put a default in help, but we don't have any default. I am not sure we can put a default email. Imho it's better to leave it blank? Or it's mandatory to have one?

Reply:
In this case I wouldn't include the default in the help.

Reply:
Yes, that's how it was initially. Ok.